packages/fscomponents/src/components/FilterList/FilterListDrilldown.tsx

```diff
@@ -357,24 +428,55 @@ export class FilterListDrilldown extends PureComponent<
           <View style={S.arrowContainer}>
             <View style={[S.arrow, S.arrowBack]} />
           </View>
-          <Text style={S.secondLevelTitle}>{item.title}</Text>
+          <Text
+            style={[S.secondLevelTitle, this.props.secondLevelTitleStyle]}
+          >
+            {this.props.secondLevelTitle || item.title}
+          </Text>
+          {this.props.onClose && this.props.secondLevelShowClose ? (
+            <TouchableOpacity
+              style={[S.rightButton, this.props.closeButtonStyle]}
+              onPress={this.handleClose}
+              accessibilityRole={'button'}
+              accessibilityLabel={FSI18n.string(componentTranslationKeys.done)}
+            >
+              <Image
+                source={this.props.closeIcon || closeIcon}
+                style={this.props.closeButtonImageStyle}
+              />
```

Comment:
Since we're setting a default image, we need to set some sizes otherwise the image will be 0x0.